pkgs/development/compilers/go/1.15.nix

```diff
@@ -12,7 +12,15 @@ let
   inherit (lib) optionals optionalString;
 
   go_bootstrap = callPackage ./bootstrap.nix {
-    inherit Security;
+    version = "1.14.15"; # Using prior version
```

Comment:
Does it need to be the prior version and is it documented anywhere?

Reply:
It's what Rust does for Rust reasons, but I don't think we have to do it for Go:

* https://github.com/NixOS/nixpkgs/commit/ba9c71b999b1

Happy to change to the same version for simplicity, let me know what you think.

Reply:
Amended to use the same version for now.

Reply:
I can't find anything that says we can't use the same version. It does make adding support for new arches easier.